lib/Refactoring/LocalRename.cpp

```diff
@@ -195,6 +195,7 @@ swift::ide::getRenameInfo(ResolvedCursorInfoPtr cursorInfo) {
 
 class RenameRangeCollector : public IndexDataConsumer {
   StringRef usr;
+  SmallString<64> displayName;
```

Comment:
Any reason not to treat this the same as `usr`? ie. local smallstring and copy into string storage? I haven't looked at where `usr` is used and if there's a good reason we're doing that anyway, so this really is a genuine question and not just rhetorically asking you to change it 😅

Reply:
I was writing a whole paragraph about how `usr` is returned and thus needs to be in `stringStorage` anyway and how `name` is only used locally so no need to waste space in `stringStorage` for it, only that I now realized that `name` is returned and `usr` is not. 😅

Changed both to use `SmallString<64>` as a member of `RenameRangeCollector` because it simplifies the code.